contracts/Vault.vy

```diff
@@ -235,10 +235,10 @@ def initialize(
     token: address,
     governance: address,
     rewards: address,
-    management: address,
     nameOverride: String[64],
     symbolOverride: String[32],
     guardian: address = msg.sender,
+    management: address = ZERO_ADDRESS,
```

Comment:
Can do `msg.sender` like `guardian`. There's no harm with it being set to the `Registry`, just requires more updates by governance.
```suggestion
    management: address = msg.sender,
```